test/runtests.jl

```diff
@@ -1,3 +1,15 @@
+# Confidence intervals should be tested component-wise against
+# expected results; splatting them into arrays and checking for
+# approximate array equality is incorrect when either limit is
+# infinite.
+macro test_ci_approx(x::Expr, y::Expr)
+    @eval begin
+        Base.Test.@test typeof($x) <: Tuple{Real,Real}
+        Base.Test.@test typeof($x) == typeof($y)
+        Base.Test.@test all(map(isapprox, $x, $y))
+    end
+end
+
```

Comment:
A good way to think about macros is that they take an expression and return another expression.  Evalings stuff inside a macro is almost always the wrong thing to do.